biome.json

```diff
@@ -11,6 +11,7 @@
       "website/.next/**",
       "website/.pages/**",
       "site/.astro/**",
+      "**/*.astro",
```

Comment:
Biome doesn't work on full Astro files, only on the frontmatter (JavaScript) section. However, it conflicts with the Astro formatter there, so I'm disabling it for now.